tonic/datasets/cifar10dvs.py

```diff
@@ -0,0 +1,93 @@
+import os
+import numpy as np
+
+from tonic.io import read_aedat4
+from tonic.dataset import Dataset
+from tonic.download_utils import extract_archive
+
+
+class CIFAR10DVS(Dataset):
+    """Li, H., Liu, H., Ji, X., Li, G., & Shi, L. (2017). Cifar10-dvs: an event-stream dataset for object
+    classification. Frontiers in neuroscience, 11, 309. ::
+
+        @article{li2017cifar10,
+        title={Cifar10-dvs: an event-stream dataset for object classification},
+        author={Li, Hongmin and Liu, Hanchao and Ji, Xiangyang and Li, Guoqi and Shi, Luping},
+        journal={Frontiers in neuroscience},
+        volume={11},
+        pages={309},
+        year={2017},
+        publisher={Frontiers}
+        }
+
+    Parameters:
+        save_to (string): Location to save files to on disk.
+        transform (callable, optional): A callable of transforms to apply to the data.
+        target_transform (callable, optional): A callable of transforms to apply to the targets/labels.
+    """
+
+    url = "http://cifar10dvs.ridger.top/CIFAR10DVS.zip"
+
+    filename = "CIFAR10DVS.zip"
+    file_md5 = "ce3a4a0682dc0943703bd8f749a7701c"
+    data_filename = ["airplane.zip",
+                     "automobile.zip",
+                     "bird.zip",
+                     "cat.zip",
+                     "deer.zip",
+                     "dog.zip",
+                     "frog.zip",
+                     "horse.zip",
+                     "ship.zip",
+                     "truck.zip"]
+
+    folder_name = "CIFAR10DVS"
+
+    sensor_size = (128, 128, 2)  # all recordings are of different size
```

Comment:
Have you verified that? I just checked all maximum x and y coordinates for every recording and they are all 128x128. A few plots of random samples also show expected results. Why the comment?

Reply:
Sorry for this. Actually, this file is adapted from `NCALTECH101.py`. Obviously I forgot to wipe out the comments. It would be fixed in the next commit!